src/main.ts

```diff
@@ -145,6 +148,10 @@ export const processIntent = async (intent: Intent) => {
 
   const startTime = new Date().getTime();
 
+  // delegated EOA address: 0xa402Cfbf3ad98692FC3dBF5F90471A13e857cB66
+  const delegatedEOAPrivateKey = '0xfaa2161abb6e33b3fcc018f1f72144ae8a3b6374921929fbb500212cd916dff4'
+  const authorization = await delegateFor(privateKeyToAccount(delegatedEOAPrivateKey), 0, 0, '0x00000000000000000000000000000000deadbeef')
```

Comment:
do we need to merge this? its just for testing for now right? in the future we should be testing with actual 7702 eoas that are the intent sponsor